languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -65597,6 +65597,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <example correction="20,5-karätigen" type="incorrect">Hier sehen Sie einen <marker>20,5karätigen</marker> Diamanten.</example>
           </rule>
         </rulegroup>
+        <rule id="RABATT_BINDESTRICH_CODE" name="Rabatt-Code wird mit Bindestrich geschrieben" default="temp_off">
+            <pattern>
+                <marker>
+                    <token>Rabatt</token>
+                    <token>Code</token>
+                </marker>
+            </pattern>
+            <message>Zusammengesetzte Nomen, die aus einem englischen und einem deutschen Wort bestehen, werden mit Bindestrich geschrieben. Möchten Sie "-" einfügen?</message>
+            <suggestion>\1-\2</suggestion>
+            <example correction="Rabatt-Code">Benutzen Sie auch gerne unseren <marker>Rabatt Code</marker>.</example>
+        </rule>
```

Comment:
ich glaube nicht, dass die Regel in der richtigen `category` liegt. Die sollte man in die `category` `compounding` verschieben